### PR TITLE
gtfs2ntfs: read route_desc into comments

### PR DESCRIPTION
**Pull request by Tristramg**
Internal issue: https://jira.kisio.org/browse/ND-1075

### CODE REVIEW

File: src/gtfs/read.rs [hunk 1]
```diff
@@ -631,6 +631,26 @@ fn manage_comment_from_stop(
     comment_links
 }
 
+fn manage_comment_from_route(
+    comments: &mut CollectionWithId<objects::Comment>,
+    route: &Route,
+) -> CommentLinksT {
```
Comment (woshilapin) [inline src/gtfs/read.rs:637]: I know we have a lot of examples of non-pure functions in the code but do you think it'd be possible to have a pure-function here that may only return a `Option<Comment>`? Something like this.

```rust
fn get_route_comment(route: &Route) -> Option<Comment> {
   // omitted
}
```

File: src/gtfs/read.rs [hunk 2]
```diff
@@ -631,6 +631,26 @@ fn manage_comment_from_stop(
...
+    if let Some(desc) = &route.desc {
+        let comment_id = "route:".to_string() + &route.id;
+        let comment = objects::Comment {
+            id: comment_id.clone(),
+            comment_type: objects::CommentType::Information,
+            label: None,
+            name: desc.to_string(),
+            url: None,
+        };
+        comments.push(comment).unwrap();
```
Comment (woshilapin) [inline src/gtfs/read.rs:648]: Please try to avoid `.unwrap()` or at least comment about why this will never panic.

**Review by woshilapin — COMMENTED** [review]
Wasn't it possible to only add a new comment on existing features instead of adding an entire new set of fixtures?

File: src/gtfs/read.rs (hunk 2 above)
Comment (Tristramg) [reply src/gtfs/read.rs]: I used an expect, it doesn’t seem common practice, but it does both the work of documenting and unwrapping. And if the unpossible happens, we’ll have some hint.

File: src/gtfs/read.rs (hunk 1 above)
Comment (Tristramg) [reply src/gtfs/read.rs]: Way nicer!
I’m adding a commit to handle the same way the stop comments (I just copied-paste…)

File: src/gtfs/read.rs (hunk 2 above)
Comment (woshilapin) [reply src/gtfs/read.rs]: Yes, `.expect()` or document is good (actually, I should push for the use of `.expect()` now so thanks :smile:)

**Comment by Tristramg** [comment]
Concerning the fixtures, many tests rely on the output of minimal. This would require small changes a bit everywhere. Do you prefer that approach?

**Comment by woshilapin** [comment]
> Concerning the fixtures, many tests rely on the output of minimal. This would require small changes a bit everywhere. Do you prefer that approach?

I see, if the alternative is to modify `minimal`, then I guess you made it right. Keep what you've done then.

**Review by woshilapin — DISMISSED** [review]

**Comment by Tristramg** [comment]
I added a commit which applies the same behaviour for stop comments.
At this point I’m wondering if we shouldn’t add a trait WithDescription to the GTFS model, but I guess I should leave the rabbit hole for now :(

**Review by woshilapin — APPROVED** [review]
I'm just wondering if `generate_route_comment` wouldn't be more accurate as the function name but apart from that, this PR is good to go.

**Comment by Tristramg** [comment]
Thanks, I renamed it

**Review by woshilapin — APPROVED** [review]

**Pull request merged**